p2p_wallet/Scenes/Main/Send/RecipentSearch/RecipientSearchView.swift

```diff
@@ -82,6 +86,36 @@ struct RecipientSearchView: View {
         }
     }
 
+    var actionButtons: some View {
+        HStack(spacing: 8) {
+            TextButtonView(
+                title: L10n.scanQR,
+                style: .primary,
+                size: .large,
+                leading: .qrCode,
+                onPressed: {
+                    UIApplication.shared.keyWindow?.endEditing(true)
```

Comment:
there is isFirstResponder property in view model, will it be better to use it instead of this force method?

Reply:
Good point, thank you!